sale_order_invoicing_finished_task/models/project.py

```diff
@@ -59,8 +59,8 @@ def _check_sale_line_state(self, sale_line_id=False):
         if sale_line_id:
             sale_lines |= self.env['sale.order.line'].browse(sale_line_id)
         for sale_line in sale_lines:
-            if (sale_line.state in ('done', 'cancel') or
-                    sale_line.invoice_status == 'invoiced'):
+            if (sale_line.state in ('done', 'cancel')
+                    or sale_line.invoice_status == 'invoiced'):
```

Comment:
Please don't change this style for nothing. Leave it the one it has.

Reply:
This change was made to follow this rule:
https://lintlyci.github.io/Flake8Rules/rules/W504.html